src/api/solr-query.js

```diff
@@ -66,7 +66,7 @@ const requestField = (fields) => fields
 
 const dateRangeFacetFieldValue = (dateRangeFacetField) => {
   var filters = dateRangeFacetField.value;
-  if (filters == null || filters.length == 0) {
+  if (!filters || filters.length === 0) {
```

Comment:
This is a small bug we didn't notice in the previous PRs, although his bug wouldn't influence our current project. The react suggests we use `===` rather than `==`.  And `filters` might become `undefined` in some cases.

Reply:
Thanks @qgan7125 this looks good!